<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-OFMQTK0O" failures="0" tests="4" name="testCases.FlipkartResulTest" time="4.546" errors="1" timestamp="2025-07-14T11:39:02 IST" skipped="3">
  <testcase classname="testCases.FlipkartResulTest" name="openApp" time="4.546">
    <error message="null value in entry: url=null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: null value in entry: url=null
at com.google.common.collect.CollectPreconditions.checkEntryNotNull(CollectPreconditions.java:33)
at com.google.common.collect.SingletonImmutableBiMap.<init>(SingletonImmutableBiMap.java:43)
at com.google.common.collect.ImmutableBiMap.of(ImmutableBiMap.java:83)
at com.google.common.collect.ImmutableMap.of(ImmutableMap.java:130)
at org.openqa.selenium.remote.DriverCommand.GET(DriverCommand.java:191)
at org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:304)
at baseClass.LibraryClass.openApplication(LibraryClass.java:89)
at baseClass.LibraryClass.openApp(LibraryClass.java:32)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:657)
at org.testng.TestRunner.beforeRun(TestRunner.java:646)
at org.testng.TestRunner.run(TestRunner.java:612)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- openApp -->
  <testcase classname="testCases.FlipkartResulTest" name="resultClick" time="0.000">
    <skipped/>
  </testcase> <!-- resultClick -->
  <testcase classname="testCases.FlipkartResulTest" name="changeWndow" time="0.000">
    <skipped/>
  </testcase> <!-- changeWndow -->
  <testcase classname="testCases.FlipkartResulTest" name="getProductDetails" time="0.000">
    <skipped/>
  </testcase> <!-- getProductDetails -->
</testsuite> <!-- testCases.FlipkartResulTest -->
